src/components/captions-control/captions-control-mini.tsx

```diff
@@ -0,0 +1,112 @@
+import {createPortal, useEffect} from 'preact/compat';
+import {registerToBottomBar} from '../bottom-bar';
+import {redux, ReservedPresetNames, style} from '../../index';
+import {Icon, IconType} from '../icon';
+import {h, VNode} from 'preact';
+import {ButtonControl} from '../button-control';
+import {Tooltip} from '../tooltip';
+import {Button} from '../button';
+import {CVAAOverlay} from '../cvaa-overlay';
+import {useState} from 'preact/hooks';
+import {connect} from 'react-redux';
+import {Text} from 'preact-i18n';
+import {CaptionsMenu} from '../captions-menu';
+import {SmartContainer} from '../smart-container';
+import {CaptionsControl} from './captions-control';
+
+const mapStateToProps = state => ({
+  controlsToMove: state.bottomBar.controlsToMove
+});
+
+const COMPONENT_NAME = 'CaptionsControlMini';
+const CaptionsControlMini = connect(mapStateToProps)((props, context) => {
+  const [shouldRender, setShouldRender] = useState(false);
+
+  const {player} = context;
+  const {controlsToMove} = props;
+  const targetId: HTMLDivElement | Document = (document.getElementById(player.config.targetId) as HTMLDivElement) || document;
```

Comment:
think of a better name, since the variable holds a div and not an id.
we should actually have a utility function for this, i don't think components should try to access the div / dom by themselves.
not crucial here though. just for the name to be correct.